server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -1846,4 +1846,19 @@ public void test_generated_sub_column_provided_null_is_validated() {
         execute("insert into t (id, obj) (select 1, {\"a\" = {\"b\" = null}} from sys.cluster)");
         assertThat(response.rowCount()).isEqualTo(0L);
     }
+
+    /*
+     * https://github.com/crate/crate/issues/14277
+     */
+    public void test_insert_into_aggregations() throws InterruptedException {
```

Comment:
We already have a similar test `testInsertFromQueryGlobalAggregate`, I wonder why this doesn't fail?

Reply:
This breaks as soon as you remove the `count(*)` with the same failure (null) e.g.:
``insert into aggs (c, s) (select sum(position), sum(position) from locations)``

Reply:
We basically only have the problem `PlanHint.PREFER_SOURCE_LOOKUP` comes into play.